src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -185,6 +185,7 @@ Copyright (C) 2011-2012 Xamarin. All rights reserved.
   <AvailableItemName Include="AndroidJavaLibrary" />
   <AvailableItemName Include="AndroidJavaSource" />
   <AvailableItemName Include="AndroidLintConfig" />
+  <AvailableItemName Include="AndroidRuleSet" />
```

Comment:
We also have some kind of lint support in the build system via `$(AndroidLintEnabled)`: https://github.com/xamarin/xamarin-android/blob/f7ea4a3db003da7be85d34eff4813c6052368e90/src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets#L1097-L1111

Does this need to be incorporated here?

Reply:
Our stuff isn't really exposed outside of our immediate designer space right now. But, this may be a good future thing.